### ISSUE TITLE
Caption doesn't stay on same page as png if png is large

### ISSUE THREAD

**Issue by robert-git** [asciidoctor/asciidoctor-pdf#1803]
If a png has a large enough height, the caption for the png doesn't stay on the same page as the image.

Similar to this issue? https://github.com/asciidoctor/asciidoctor-pdf/issues/690

asciidoctor-pdf version:
```
asciidoctor-pdf --version
Asciidoctor PDF 1.5.3 using Asciidoctor 2.0.10 [https://asciidoctor.org]
Runtime Environment (ruby 2.5.8p224 (2020-03-31 revision 67882) [x86_64-linux]) (lc:UTF-8 fs:UTF-8 in:UTF-8 ex:UTF-8)
```

To reproduce the issue, create an adoc file with the following content:
```
= Test If Caption Stays On Same Page As Large Image

<<<

.Some Image
image::long activity diagram.png[align=center]

```
Image file "long activity diagram.png":
![image](https://user-images.githubusercontent.com/3901138/96685430-b62b6a80-1342-11eb-9947-28bf8cfdba38.png)

Resulting pdf:
[Test If Caption Stays On Same Page As Large Image.pdf](https://github.com/asciidoctor/asciidoctor-pdf/files/5413939/Test.If.Caption.Stays.On.Same.Page.As.Large.Image.pdf)

**Comment by mojavelinux** [comment]
This is a known issue with 1.5.3 and has been fixed in the upcoming 2.0.0 release. I'm not sure which commit fixed it, but I just tested to confirm that it is fixed in master.

**Comment by mojavelinux** [comment]
Since I'm not sure which commit fixed the problem, I probably won't backport this to 1.5.x. However, if someone figures out which commit it is, I will accept a PR to that branch.

**Comment by mojavelinux** [comment]
I'm going to mark this as a regression in 1.5.x, but do note that it does not affect master.

**Comment by mojavelinux** [comment]
It was this commit: d5cef8772ca5cec4df49660ff10ab2c836a55923

**Comment by mojavelinux** [comment]
Resolved by #1862.
